doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -974,32 +974,42 @@ Generation of induction principles with ``Scheme``
    .. insertprodn scheme_kind sort_family
 
    .. prodn::
-      scheme_kind ::= Equality for @reference
-      | {| Induction | Minimality | Elimination | Case } for @reference Sort @sort_family
-      sort_family ::= Set
-      | Prop
+      scheme_kind ::= {| Induction | Minimality | Elimination | Case } for @reference Sort @sort_family
+      sort_family ::= Prop
       | SProp
+      | Set
       | Type
 
-  A high-level tool for automatically generating
-  (possibly mutual) :term:`induction principles <induction principle>`
-  for given types and sorts.
-  Each :n:`@reference` is a different inductive type identifier belonging to
-  the same package of mutual inductive definitions.
-  The command generates the :n:`@ident`\s as mutually recursive
-  definitions. Each term :n:`@ident` proves a general principle of mutual
-  induction for objects in type :n:`@reference`.
+   Generates an :term:`induction principles <induction principle>` with given
+   :n:`scheme_type`\s and :n:`scheme_sort`\s for an inductive type. In the case
```

Comment:
```suggestion
   Generates :term:`induction principles <induction principle>` with given
   *scheme type* and :n:`@sort_family`\s for an inductive type. In the case
```

```suggestion
   *scheme type* and :n:`sort_family`\s for an inductive type. In the case
```

Reply:
> How can I add scheme_type to the grammar(?)

That's not supported--the concept of the tool is to guarantee that the grammar of commands, tactics and insertprodn matches the source code.  You would have to not splice scheme_type.